src/pencilkit.cs

```diff
@@ -30,14 +32,16 @@
 
 namespace PencilKit {
 
-	[iOS (13, 0), NoMac]
+	[iOS (13, 0), Mac (10, 16)]
+	[Introduced (PlatformName.MacCatalyst, 14, 0)]
```

Comment:
Not sure what we are doing for Catalyst support - are we adding availability attributes now or later?

Reply:
Is fine to add them now, won't hurt and it is likely to help us in the future